.github/workflows/trufflehog.yml

```
@@ -0,0 +1,48 @@
+name: TruffleHog Secrets Scan
+on: [pull_request]
+jobs:
+  TruffleHog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install TruffleHog
+        run: |
+          wget -q https://github.com/trufflesecurity/trufflehog/releases/download/v3.54.1/trufflehog_3.54.1_linux_amd64.tar.gz
+          tar xzf trufflehog_3.54.1_linux_amd64.tar.gz
+          sudo cp trufflehog /usr/local/bin
```

```suggestion
          sudo tar xzf trufflehog_3.54.1_linux_amd64.tar.gz --directory=/usr/local/bin/ trufflehog
```
nit: combines the two commands in one.